examples/layers/JSONLayers/FL_Buildings.json

```diff
@@ -0,0 +1,19 @@
+{
+    "url"       : "http://wxs.ign.fr/72hpsel8j8nhb5qgdh07gcyp/geoportail/wfs?",
+    "protocol"  : "wfs",
+    "version"   : "2.0.0",
+    "axisOrder" : "inverted",
+    "id"        : "wfsBuilding",
+    "title"     : "BDTOPO_BDD_WLD_WGS84G:bati_remarquable,BDTOPO_BDD_WLD_WGS84G:bati_indifferencie,BDTOPO_BDD_WLD_WGS84G:bati_industriel",
+    "styleName" : "",
+    "extractStyle" : true,
+    "level"     : 16,
```

Comment:
I don't know, but the layer can hold options not needed by the protocol.
In this case, level and bbox are used by `FeatureProcessing`.